test/functional/fixtures/api/es-next/take-screenshot/test.js

```diff
@@ -50,6 +50,8 @@ const getReporter = function (scope) {
     });
 };
 
+const itFn = config.proxyless ? it.skip : it;
```

Comment:
The project has this condition in other tests. Can you move it to utils and replace `if (!config.proxyless)` everywhere?